samples/files.js

```diff
@@ -312,6 +313,79 @@ async function generateSignedUrl(bucketName, filename) {
   // [END storage_generate_signed_url]
 }
 
+async function generateV4ReadSignedUrl(bucketName, filename) {
+  // [START storage_generate_signed_url_v4]
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const filename = 'File to access, e.g. file.txt';
+
+  // These options will allow temporary read access to the file
+  const options = {
+    version: 'v4',
+    action: 'read',
+    expires: Date.now() + 1000 * 60 * 60, // one hour
+  };
+
+  // Get a v4 signed URL for reading the file
+  const [url] = await storage
+    .bucket(bucketName)
+    .file(filename)
+    .getSignedUrl(options);
+
+  console.log('Generated GET signed URL:');
+  console.log(url);
```

Comment:
this is a little wordy, I might be tempted to consolidate in a single template string comment:

```
console.info(`generated the signed URL ${url} which is valid for one hour`)
```

Reply:
Good call out, but as @frankyn mentioned, we have standardized the outputs of these samples across languages so we'd just have to follow the canonical :)